### ISSUE TITLE
both genetic-impact correlation and genetic-effect correlation greater than 1

### ISSUE THREAD

**Issue by gunns2** [brielin/Popcorn#33]
Hi, 
After running popcorn we are finding that both the genetic-impact correlation and genetic-effect correlation are greater than 1, which was unexpected. Do you think this is the result of a bug or some error on our part? If not, how should we interpret this value?

thank you!!

**Comment by brielin** [comment]
hi, this is fine. the default regression model is unbounded, so it is possible to get estimates above 1. Generally the 95% confidence interval should contain 1 in this case. If you use the MLE model, the estimate will be bounded by +/- 1.

**Comment by gunns2** [comment]
thank you so much!

**Comment by estelabruxel** [comment]
Hello

I’ve been running Popcorn for different traits across different ancestries.
Some results are OK and similar from previous, but others I got correlation greater than 1, meanly for AFR-AMR ancestries. I saw here the same case, but yet if I use MLE model, it is still greater than 1. How I can interpret it? 

Another thing is that for one trait I got a negative result. How I can handle with it? I’ve already checked  the alleles in BED files are according to FASTA files to generate the LD scores. 

Best,
Estela
